src/lib/components/labeledInput/labeledInput.test.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import LabeledInput from './labeledInput.js';
+
+describe('LabeledInput', () => {
+  it('renders properly', () => {
+    const tree = renderer
+      .create(<LabeledInput name="testing" value="" label="Testing input" />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders properly with required', () => {
+    const tree = renderer
+      .create(<LabeledInput name="testing" value="" label="Testing required input" required />)
```

Comment:
Remember that we are not going to use the `required`

Reply:
Ok, anyway I leave this test here because if we use required it will work too, and it's already implemented